chpl/chpl-service/src/main/java/gov/healthit/chpl/dao/statistics/SurveillanceStatisticsDAO.java

```diff
@@ -6,18 +6,30 @@
 
 import javax.persistence.Query;
 
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Repository;
 
 import gov.healthit.chpl.dao.impl.BaseDAOImpl;
+import gov.healthit.chpl.domain.surveillance.NonconformityClassification;
 import gov.healthit.chpl.dto.CertificationCriterionDTO;
 import gov.healthit.chpl.dto.NonconformityTypeStatisticsDTO;
-import gov.healthit.chpl.entity.surveillance.NonconformityAggregatedStatisticsEntity;
```

Comment:
Is this class/table used anywhere else? I did a quick search in the code and didn't find anything, but I may have missed it. Wondering if it can be deleted?